### PR TITLE
replace document proto with multimodal document

### PR DESCRIPTION
**Pull request by bwanglzu**
1. delete unused notebook.
2. replace `documentproto` with `MultimodalDocument`
3. refactor `index` logic since we need to set `mime_type` for the document.

Tested, index & query worked.

### CODE REVIEW

File: multimodal-search-tirg/app.py [hunk 1]
```diff
@@ -60,17 +59,14 @@ def print_result(resp):
...
-        chunk2.text = text
-        yield doc
+        with Document() as chunk1:
+            chunk1.modality = 'image'
+            with open(image_path, 'rb') as fp:
+                chunk1.buffer = fp.read()
+        with Document() as chunk2:
+            chunk2.modality = 'text'
+            chunk2.text = text
+        yield MultimodalDocument(chunks=[chunk1, chunk2])
```
Comment (JoanFM) [inline multimodal-search-tirg/app.py:69]: can we try constructing with modality_mapping as input?

**Review by JoanFM — COMMENTED** [review]
have u done some minor test?

**Review by JoanFM — APPROVED** [review]

**Comment by bwanglzu** [comment]
@JoanFM I build the docker image locally, succeed, still downloading the data and run indexing & query, one moment.

File: multimodal-search-tirg/app.py [hunk 2]
```diff
@@ -57,18 +58,27 @@ def print_result(resp):
...
+            doc.mime_type = 'image/jpeg'
+        yield doc
+
 def query_generator(image_paths, text_queries):
     for image_path, text in zip(image_paths, text_queries):
         with open(image_path, 'rb') as fp:
             buffer = fp.read()
-        yield MultimodalDocument(modality_content_map={'image': buffer, 'text': text})
+        with MultimodalDocument(modality_content_map={'image': buffer, 'text': text}) as md:
+            md.mime_type = 'image/jpeg'
```
Comment (JoanFM) [inline multimodal-search-tirg/app.py:73]: why multimodal has a mime type? it should have none

**Review by JoanFM — CHANGES_REQUESTED** [review]

File: multimodal-search-tirg/app.py (hunk 2 above)
Comment (bwanglzu) [reply multimodal-search-tirg/app.py]: removed

**Review by JoanFM — APPROVED** [review]

**Pull request merged**